src/lib/searchUtilActions/searchClient.ts

```diff
@@ -1,74 +0,0 @@
-'use client';
```

Comment:
searchClient.ts and searchServer.ts files are still there but empty

Reply:
You are right, done